service/src/main/resources/db/changesets/20240104.yaml

```diff
@@ -0,0 +1,20 @@
+# Change Jobs table to ImputationJobs table
+databaseChangeLog:
+  - changeSet:
+      id: change jobs to imputation_jobs and remove fields that Stairway tracks
+      author: mma
+      changes:
+        - renameTable:
+            oldTableName: jobs
+            newTableName: imputation_jobs
+        - dropColumn:
+            tableName: imputation_jobs
+            columns:
+            - column:
+                name: pipeline_id
+            - column:
+                name: time_submitted
+            - column:
+                name: time_completed
+            - column:
+                name: status
```

Comment:
nice! we can clean up the pipelines table here if we want.

Reply:
good call, done!